rotkehlchen/db/upgrades/v39_v40.py

```diff
@@ -119,15 +126,19 @@ def _purge_kraken_events(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _reset_kraken_events')
 
 
-def _add_base_chain_data(write_cursor: 'DBCursor') -> None:
-    log.debug('Enter _add_base_chain_data')
-    write_cursor.execute('INSERT OR IGNORE INTO location(location, seq) VALUES ("j", 42);')
+def _add_new_supported_chains_data(write_cursor: 'DBCursor') -> None:
+    log.debug('Enter _add_new_supported_chains_data')
+    write_cursor.executemany(
+        'INSERT OR IGNORE INTO location(location, seq) '
+        'VALUES (?, ?)',
+        (('j', 42), ('k', 43)),
+    )
```

Comment:
if we use the remote data this is not needed. You will need to create a PR like this one https://github.com/rotki/data/pull/27

Reply:
Ok, I created a PR in the data repo that adds the gnosis rpc nodes: https://github.com/rotki/data/pull/28
I removed the addition of rpc nodes from the 39_40 upgrade because they are added by the data migration 12. The new locations are not added by the data_migration so they are added by the upgrade.